common/changes/@typespec/compiler/fix-formatting-doc-directive_2023-09-19-17-39.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/compiler",
+      "comment": "Fix: Formatting of comments between a directive or doc and its node",
```

```suggestion
      "comment": "Fix: Correct formatting of comments between a directive or doc and its node",
```
Fix being a tag, the description is missing a verb. Pick any you like 😁